tox.ini

```diff
@@ -41,7 +41,7 @@ commands =
     python setup.py check -m -r -s -v
 
 deps =
-    docutils
+    docutils==0.12
```

Comment:
UPDATE: I filed this as [`docutils` bug #302](https://sourceforge.net/p/docutils/bugs/302/).